stories/modules-tagmanager-setup.stories.js

```diff
@@ -49,7 +49,51 @@ import {
 import { MODULES_ANALYTICS } from '../assets/js/modules/analytics/datastore/constants';
 import * as fixtures from '../assets/js/modules/tagmanager/datastore/__fixtures__';
 import * as modulesFixtures from '../assets/js/googlesitekit/modules/datastore/__fixtures__';
-import { parseLiveContainerVersionIDs } from '../assets/js/modules/tagmanager/datastore/__factories__/utils';
+import {
+	createBuildAndReceivers,
+	parseLiveContainerVersionIDs,
+} from '../assets/js/modules/tagmanager/datastore/__factories__/utils';
+
+function buildAndReceiveProperties( registry, accountID, internalContainerID ) {
```

Comment:
This is a bit confusing as named as it's not receiving properties (GA/Search Console entities) but building and receiving GTM containers with properties configured.

It's also a bit odd that it seems to depend on the existing live container version.

Let's inline this utility as it's a bit hard to understand by itself and the part below is already quite minimal.

Reply:
Updated ✔️